src/models/User.js

```diff
@@ -46,13 +46,17 @@ class User extends Resource {
   }
 
   /**
+   * @param {object} queryParameters
    * @returns {Promise<User>}
    */
-  update() {
-    return this.client.updateUser(this.id, this);
+  update(queryParameters) {
+    return this.client.updateUser(this.id, this, queryParameters);
   }
-  delete() {
-    return this.client.deactivateOrDeleteUser(this.id);
+  /**
+   * @param {object} queryParameters
+   */
+  delete(queryParameters) {
+    return this.client.deactivateOrDeleteUser(this.id, queryParameters);
```

Comment:
same question here, breaking changes?

Reply:
No issues with adding a parameter, they can just be omitted